inline-translation/css/custom.css

```
@@ -162,3 +162,14 @@ iframe.translator-untranslatable {
 	margin-bottom: .2em;
 	cursor: pointer;
 }
+.webui-popover a.inline-placeholder{
+	background: #F1ADAD;
+	padding: 0.3em;
+	margin-right: 0.2em;
+	border-radius: 2px;
+	color: #fff;
+	text-decoration: none;
```

I will suggest you to add a line though to this CSS, so the people with color blindness can differentiate between the existing and non-existing placeholders:
```suggestion
	text-decoration: line-through;
```